hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -207,46 +220,41 @@ public void createContract(
             @NonNull final ContractCreateTransactionBody body,
             final long nonce,
```

Comment:
is the nonce used?

Reply:
Great point, we already set the initial nonce [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomContractCreationProcessor.java#L82) in `CustomContractCreationProcessor`. 

Removed the `nonce` argument from both methods.